libimage/runtime.go

```diff
@@ -515,20 +516,29 @@ func (r *Runtime) imageReferenceMatchesContext(ref types.ImageReference, options
 		return false, err
 	}
 
+	writeMessage := func(msg string) {
+		if writer == nil {
+			logrus.Debugf("WARNING: %s", msg)
```

Comment:
Why would this be at debug level if you print a warning?

Reply:
Fair point.